.circleci/config.yml

```diff
@@ -3,17 +3,16 @@ jobs:
   build:
     working_directory: /home/circleci/app
     docker:
-    - image: circleci/python:3.7-buster-browsers
```

Comment:
if we remove `-browsers` will that affect the frontend tests? Or is that overwritten below in line 55?